PURL-TYPES.rst

```diff
@@ -59,6 +59,26 @@ apk
       pkg:apk/alpine/curl@7.83.0-r0?arch=x86
       pkg:apk/alpine/apk@2.12.9-r3?arch=x86
 
+bazel
+-----
+``bazel`` for Bazel modules as specified at ``https://bazel.build/external/module``:
+
+- The default repository is the Bazel Central Registry (BCR) ``https://bcr.bazel.build``.
+- The ``name`` is the module name.
+- The ``version`` is the module version.
+- The optional ``repository_url`` can be used to specify the URL of an
```

Comment:
This is a standard qualifier and option, so there is no need to repeat this here